specification/loadtestservice/LoadTestService.Management/routes.tsp

```diff
@@ -0,0 +1,85 @@
+import "./models.tsp";
+import "./customOperation.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@@encodedName(LoadTests.createOrUpdate::parameters.resource,
+  "application/json",
+  "LoadTestResource"
+);
+
+@@encodedName(LoadTests.update::parameters.properties,
+  "application/json",
+  "LoadTestResourcePatchRequestBody"
+);
+
+@@encodedName(Quotas.checkAvailability::parameters.body,
+  "application/json",
+  "QuotaBucketRequest"
+);
+
+@armResourceOperations
+interface LoadTests {
+  listBySubscription is ArmListBySubscription<LoadTestResource>;
+
+  listByResourceGroup is ArmResourceListByParent<LoadTestResource>;
+  get is ArmResourceRead<LoadTestResource>;
+
+  createOrUpdate is ArmResourceCreateOrUpdateAsync<
+    LoadTestResource,
+    LroHeaders = ArmAsyncOperationHeader
+  >;
+
+  update is ArmResourcePatchAsync<
+    LoadTestResource,
+    LoadTestProperties,
+    LroHeaders = ArmAsyncOperationHeader
+  >;
+
+  delete is ArmResourceDeleteWithoutOkAsync<LoadTestResource>;
+
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "OperationID already published."
+  @operationId("LoadTests_ListOutboundNetworkDependenciesEndpoints")
+  @doc("Lists the endpoints that agents may call as part of load testing.")
+  @extension(
```

Comment:
nit: Is this needed? The Page<T> on response should add this already.

Reply:
Removed